### PR TITLE
add link to new documentation

### PR DESCRIPTION
**Pull request by shinobu**
This pull request adds a link to the new documentation

### CODE REVIEW

**Comment by white-gecko** [comment]
The README ist still important, so the phrase "This documentation …" is not good. It should just be a prominent link to the documentation. I've already disabled the wiki here on GitHub,

**Comment by shinobu** [comment]
oh sorry, i didn't know it was already deactivated, gonna change that in a sec

**Comment by white-gecko** [comment]
I think the link should be more prominent. Maybe you add an extra section before the Introduction.

**Comment by shinobu** [comment]
you can check it again @white-gecko

**Comment by white-gecko** [comment]
The documentation link should go to the top of the README file.
And please squash the commits.

**Comment by shinobu** [comment]
I added the link at the top, put the ##Documentation at the bottom (the notice about how they can propose changes is in my point of view important)

**Comment by white-gecko** [comment]
Please check it again! Build-Status is now linked to docs.ontowiki.net. You could also update the build status using the travis icons.

**Comment by shinobu** [comment]
I have literary no idea how that even could have happened.
Fixed it, added the Travis build status (still kept the owdev one for now), changed the look so that it resembles Erfurt

should be ok now @white-gecko

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # OntoWiki
 
-[![Build Status](http://owdev.ontowiki.net/job/OntoWiki/badge/icon)](http://owdev.ontowiki.net/job/OntoWiki/)
+
+| CI System      | Status                                                                                                           | 
+| -------------- | ---------------------------------------------------------------------------------------------------------------- |
+| Travis CI      | [![Travis CI Build Status](https://travis-ci.org/AKSW/OntoWiki.svg)](https://travis-ci.org/AKSW/OntoWiki/)           |
+| owdev Jenkins  | [![owdev Build Status](http://owdev.ontowiki.net/job/OntoWiki/badge/icon)](http://owdev.ontowiki.net/job/OntoWiki/)  |
+
+[Documentation](https://github.com/AKSW/docs.ontowiki.net)
```
Comment (pfrischmuth) [inline README.md]: Link seems to be wrong

File: README.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 # OntoWiki
 
-[![Build Status](http://owdev.ontowiki.net/job/OntoWiki/badge/icon)](http://owdev.ontowiki.net/job/OntoWiki/)
+
+| CI System      | Status                                                                                                           | 
+| -------------- | ---------------------------------------------------------------------------------------------------------------- |
+| Travis CI      | [![Travis CI Build Status](https://travis-ci.org/AKSW/OntoWiki.svg)](https://travis-ci.org/AKSW/OntoWiki/)           |
+| owdev Jenkins  | [![owdev Build Status](http://owdev.ontowiki.net/job/OntoWiki/badge/icon)](http://owdev.ontowiki.net/job/OntoWiki/)  |
```
Comment (pfrischmuth) [inline README.md]: Please remove it

**Review by pfrischmuth — CHANGES_REQUESTED** [review]

**Comment by white-gecko** [comment]
I'd like to care for this ;-)

[Travis CI Build Status](https://travis-ci.org/AKSW/OntoWiki/) [Latest Stable Version](https://packagist.org/packages/aksw/ontowiki) [Total Downloads](https://packagist.org/packages/aksw/ontowiki) [License](https://packagist.org/packages/aksw/ontowiki) [composer.lock](https://packagist.org/packages/aksw/ontowiki)


[OntoWiki User Documentation](https://docs.ontowiki.net) [OntoWiki API Documentation](http://api.ontowiki.net/)

**Comment by white-gecko** [comment]
The actual images can be seen in my comment

**Review by pfrischmuth — APPROVED** [review]
LGTM

**Comment by white-gecko** [comment]
@pfrischmuth works, it was the rawgit cache delaying the delivery of the SVG files.

**Pull request merged**